integreat_cms/core/settings.py

```diff
@@ -31,6 +31,9 @@
     "INTEGREAT_CMS_MATOMO_URL", "https://statistics.integreat-app.de"
 )
 
+#: Enable tracking of API requests in Matomo
+MATOMO_TRACKING = os.environ.get("INTEGREAT_CMS_MATOMO_TRACKING", False)
```

Comment:
```suggestion
MATOMO_TRACKING = bool(
    strtobool(os.environ.get("INTEGREAT_CMS_MATOMO_TRACKING", "False"))
)
```
If we just get the environment variable without parsing, we have the problem that setting `MATOMO_TRACKING = False` in the config file would evaluate to `True` because it's a non-empty string.

Reply:
Right, thanks :see_no_evil: